ApplicationLibCode/ProjectDataModel/Faults/RimFaultReactivationDataAccessorWellLogExtraction.h

```diff
@@ -0,0 +1,52 @@
+/////////////////////////////////////////////////////////////////////////////////
+//
+//  Copyright (C) 2023 -    Equinor ASA
+//
+//  ResInsight is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  ResInsight is distributed in the hope that it will be useful, but WITHOUT ANY
+//  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+//  FITNESS FOR A PARTICULAR PURPOSE.
+//
+//  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+//  for more details.
+//
+/////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "RimFaultReactivationDataAccessor.h"
+#include "RimFaultReactivationEnums.h"
+
+#include <vector>
+
+//==================================================================================================
+///
+///
+//==================================================================================================
+class RimFaultReactivationDataAccessorWellLogExtraction
+{
+public:
+    RimFaultReactivationDataAccessorWellLogExtraction();
+    ~RimFaultReactivationDataAccessorWellLogExtraction();
+
+    static std::pair<double, cvf::Vec3d>
+        getPorBar( const std::vector<cvf::Vec3d>& intersections, std::vector<double>& values, const cvf::Vec3d& position, double gradient );
```

Comment:
Following ResInsight nameing style, getPorBar probably should be named just porBar(...) or caclulatePorBar(...) or something?

Reply:
Went for calculatePorBar.